paasta_tools/deployment_utils.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
Do you have future plans for this file? If not it seems kind of wasteful.

There's a lot of commands in utils (and elsewhere) that are only used in service deployment. I'm personally a fan of breaking up utils.